Dockerfile

```diff
@@ -38,18 +35,17 @@ LABEL uk.gov.defra.node.node-version=$BASE_VERSION \
 
 FROM production AS development
 
-ENV NODE_ENV development
+ENV NODE_ENV=development
 
 LABEL uk.gov.defra.node.repository=defradigital/node-development
 
-# node-gyp is a common requirement for NPM packages. It must be installed as root.
+# Install common dependencies not included in the base alpine image
 USER root
-RUN apk update && \
-    apk add --no-cache git && \
-    apk add --no-cache --virtual .gyp 'python3' make 'g++'
-# Pact dependencies are not included in Alpine image for contract testing
-RUN apk add --no-cache bash wget \
-    && wget -q -O /etc/apk/keys/sgerrand.rsa.pub https://alpine-pkgs.sgerrand.com/sgerrand.rsa.pub \
-    && wget https://github.com/sgerrand/alpine-pkg-glibc/releases/download/2.35-r1/glibc-2.35-r1.apk \
-    && apk add glibc-2.35-r1.apk
+
+# node-gyp is a common requirement for NPM packages.
+RUN apk add --no-cache bash 'g++' git make 'python3'
```

Comment:
Have removed virtual group as can't find a valid use case for it and no evidence that teams are gaining an advantage from it.